Makefile

```
@@ -478,22 +477,77 @@ define KIND_ENV
 $(1): export DOCKER_REGISTRY=localhost:5000
 $(1): export LOCAL_AGENT_IMAGE=$$(DOCKER_REGISTRY)/$$(DOCKER_DEV_ACCOUNT)/cilium-dev:$$(LOCAL_IMAGE_TAG)
 $(1): export LOCAL_OPERATOR_IMAGE=$$(DOCKER_REGISTRY)/$$(DOCKER_DEV_ACCOUNT)/operator-generic:$$(LOCAL_IMAGE_TAG)
+$(1): export LOCAL_CLUSTERMESH_IMAGE=$$(DOCKER_REGISTRY)/$$(DOCKER_DEV_ACCOUNT)/clustermesh-apiserver:$$(LOCAL_IMAGE_TAG)
 endef
 
-$(eval $(call KIND_ENV,kind-image-agent))
-kind-image-agent: kind-ready ## Build cilium-dev docker image and import it into kind.
+$(eval $(call KIND_ENV,kind-clustermesh-image))
+kind-clustermesh-image: kind-ready kind-build-clustermesh-apiserver kind-build-image-agent kind-build-image-operator
```

```suggestion
kind-clustermesh-images: kind-ready kind-build-clustermesh-apiserver kind-build-image-agent kind-build-image-operator
```

Nit. Seems clearer to me.

---

Changed it.

I was also thinking about it, but it took me some time to adapt to `kind-image` as I was mistyping it quite often :joy: